tests/MatrixDotNet.PerformanceTesting/Matrix/BenchReverse.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using BenchmarkDotNet.Attributes;
+using MatrixDotNet.Extensions.Builder;
+using MatrixDotNet.Extensions.Conversion;
+
+namespace MatrixDotNet.PerformanceTesting.Matrix
+{
+    [MemoryDiagnoser]
+    public class BenchReverse : PerformanceTest
+    {
+        private Matrix<int> _matrix;
+        private Matrix<byte> _matrixByte;
+        private Matrix<float> _matrixFloat;
+        private Matrix<double> _matrixDouble;
+        
+        [GlobalSetup]
```

Comment:
May be here should be not Global but Iteration Setup?

Reply:
In this case, I think need to use  GlobalSetup this data need only initialize we compare reverse without initializing on each step